silo-to-silo/contracts/src/SiloToSilo.sol

```diff
@@ -174,9 +172,13 @@ contract SiloToSilo is AccessControl {
 
     function _stringToUint(bytes memory b) private pure returns (uint128) {
         uint128 result = 0;
+        
         for (uint256 i = 0; i < b.length; i++) {
-            result = result * 10 + (uint128(uint8(b[i])) - 48);
+            if (uint8(b[i]) > uint8(47) && uint8(b[i]) < uint8(58)) {
```

Comment:
Let's assign the `b[i]` to a variable
`uint256 v = uint256(uint8(b[i]));`

also `47` and `58` confuse me so I think it is better to use >= 48 && <=57

Reply:
fixed: https://github.com/aurora-is-near/rainbow-token-connector/pull/226/commits/82d74e3178cc9e3bb9e292cf6b286722d6338591